src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -26,6 +27,7 @@ public partial class PortViewModel : ViewModelBase
         private const double autocompletePopupSpacing = 2.5;
         private const double proxyPortContextMenuOffset = 20;
         internal bool inputPortDisconnectedByConnectCommand = false;
+        private bool nodeAutoCompleteMarkerVisible;
```

Comment:
@johnpierson guessing the failure reason - let's initialize this with false. I suspect during tests is true and the tests record the control  no

Reply:
It is because the control is a child element of the port. So now it reports back 9 for this node when it was previously 6.  
![image](https://github.com/user-attachments/assets/809d59af-0f0d-4884-9240-eb0315e0667c)